test/cli/error-url/error-url.out

```diff
@@ -1,21 +1,24 @@
-test/cli/error-url/error-url.rb:4: Unable to resolve constant `MissingConstant` https://example.com/errors/?error=5002
-     4 |    MissingConstant
-            ^^^^^^^^^^^^^^^
-    https://github.com/sorbet/sorbet/tree/master/rbi/stdlib/optparser.rbi#L743: Did you mean: `OptionParser::MissingArgument`?
-     743 |  class MissingArgument < OptionParser::ParseError; end
-            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The key change here is that the `stdlib` RBI path is not in the `.out` file anymore.

Reply:
The test itself was trying to test the `--error-url` flag.